prow/tide/gerrit.go

```diff
@@ -63,6 +63,7 @@ type gerritClient interface {
 	GetChange(instance, id string, addtionalFeilds ...string) (*gerrit.ChangeInfo, error)
 	GetBranchRevision(instance, project, branch string) (string, error)
 	SubmitChange(instance, id string, wait bool) (*gerrit.ChangeInfo, error)
+	SetReview(instance, id, revision, message string, labels map[string]string) error
```

Comment:
Nit: Should this be `SetReviewComment` ? From the PR description it sounds like Tide is going to leave a comment.